frontend/web/components/ErrorMessage.js

```diff
@@ -34,6 +34,16 @@ export default class ErrorMessage extends PureComponent {
                 .join('<br/>'),
             }}
           />
+        ) : this.props.exceeded ? (
+          <>
+            Your organisation has exceeded its API usage quota{' '}
+            <b>{`(${error}).`}</b>{' '}
+            {Utils.getPlanName(this.props.organisationPlan) === 'Free' ? (
+              <b>Please upgrade your plan to continue receiving service.</b>
+            ) : (
+              <b>Automated billing for the overages may apply.</b>
+            )}
+          </>
```

Comment:
The bolding of the error amount looks weird because the end of the sentence extends into the next one. I recommend doing it like this:
```suggestion
          <>
            Your organisation has exceeded its API usage quota{' '}
            {`(${error}).`}{' '}
            {Utils.getPlanName(this.props.organisationPlan) === 'Free' ? (
              <b>Please upgrade your plan to continue receiving service.</b>
            ) : (
              <b>Automated billing for the overages may apply.</b>
            )}
          </>
```

Reply:
Done